utils/webshop_login_helpers.py

```diff
@@ -0,0 +1,20 @@
+import time
+from utils.webshop_config import WEBSHOP_BASE_URL
+
+def webshop_login(email, password, page):
+
```

Comment:
Could benefit from the docstring like https://peps.python.org/pep-0257/#multi-line-docstrings